### ISSUE TITLE
Add pod to PerlBench.pm

### ISSUE THREAD

**Issue by briandfoy** [briandfoy/perlbench#2]
There's not much there now.

**Comment by jkeenan** [comment]
> There's not much there now.

Indeed.  I installed this distro against perl-5.30.0 using `cpanm`, then called:
```
[p5p] $ perldoc PerlBench
[p5p] $ 
```
Is this modules still supported?

Thank you very much.
Jim Keenan

**Comment by briandfoy** [comment]
This module is still supported, and this ticket is to add some docs :)

Is there something not working?

**Comment by jkeenan** [comment]
> This module is still supported, and this ticket is to add some docs :)
> 
> Is there something not working?

If there are no docs, how does anyone know how to use it?

This came up because Perl 5 got a [ticket alleging a slowdown between 5.28 and 5.30](https://github.com/Perl/perl5/issues/17558).  The assertion was made on the basis of a complicated program which was said to be based on perlbench.  I searched for that on CPAN and this distro came up before Gisle Aas's perlbench of many years ago.

Thank you very much.
Jim Keenan

**Comment by briandfoy** [comment]
This is Gisle's perlbench. I stepped up to rescue it and now it has a GitHub repo. 

Jim, you are better than this. Help or don't. I'm helping by giving this project a repo and responding to tickets. Complaining like this to someone you know who understands this is disruptive and disrespectful.
